scripts/translations.js

```diff
@@ -0,0 +1,52 @@
+const fs = require('fs');
+const path = require('path');
+
+const main = async () => {
+  let packages;
+
+  try {
+    packages = fs.readdirSync(path.resolve(process.cwd(), 'packages'), 'utf8');
+
+    packages = packages.filter((pkg) => {
+      return pkg.indexOf('strapi') !== -1;
+    });
+  } catch (error) {
+    console.error(`Can't get strapi packages`);
+  }
+
+  const packageCheckingPromises = packages.map((pkg) => {
```

Comment:
What happens if the `fs.readdirSync` throws an error because it means the `packages` variable will be set to `undefined` and it will be impossible to map on it?